packages/core/src/components/Icon/__tests__/allIcons.test.ts

```diff
@@ -7,9 +6,10 @@ import {
   getClickableScreenReaderAccessProps,
   getClickableIconScreenReaderAccessProps
 } from "../../../helpers/screenReaderAccessHelper";
+import { ArrayLastElement, Helpers } from "./helpers";
 
 type IconPath = `/icons/${keyof typeof allIcons}`;
-type SplittedIconPath = SplitString<IconPath, "/">;
+type SplittedIconPath = Helpers<IconPath, "/">;
```

Comment:
is this intended? should it be called Helpers? I don't understand what exactly the helpers type does, but maybe a more meaningful name like SplitString can better explain what it does

Reply:
I'm not actually familiar with what's going on there, but Helpers sound too generic